README.md

```diff
@@ -111,3 +111,114 @@ Want to contribute? Great!
 Just fork the github repo, do some awesome stuff and create a pull request.
 
 Report issues or feature enhancements/requests on the [Issues](https://github.com/piejanssens/premiumizer/issues) page
+
+# Setting up Sonarr/Couchpotato to use Premiumizer
+
+To utilize premiumizer as a downloader for [Sonar](https://sonarr.tv) or [CouchPotato](https://couchpota.to) style projects, configure these projects to use the Black Holes that premiumizer monitors.  For example, consider the following excerpt from a [docker-compose.yml](https://gist.github.com/drmikecrowe/43929be99a22a083aaf6421460d52d19) file:
+
+```yml
+# CouchPotato – Movie Download and Management
+  couchpotato:
+    image: "linuxserver/couchpotato"
+    hostname: couchpotato
+    container_name: "couchpotato"
+    volumes:
+      - ./docker/couchpotato:/config
+      - ./docker/Downloads/blackhole/movies:/blackhole
+      - ./docker/Downloads/completed/movies:/downloads
+      - ./NAS/Movies:/movies
+    ports:
+      - "5050:5050"
+    expose:
+      - 5050
+    restart: unless-stopped
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - UMASK_SET=002
+      - TZ=America/New_York
+      - VIRTUAL_HOST=couchpotato.${HOST_DOMAIN}
+      - VIRTUAL_PORT=5050
+
+# Sonarr – TV Show Download and Management
+  sonarr:
+    image: "linuxserver/sonarr"
+    hostname: sonarr
+    container_name: "sonarr"
+    volumes:
+      - ./docker/sonarr:/config
+      - ./docker/Downloads/blackhole/tv:/blackhole
+      - ./docker/Downloads/completed:/downloads
+      - ./NAS/TV:/tv
```

Comment:
Note: If the "Read Only" flag in the blackhole configuration is disabled, Sonarr will move (instead of copy) episode files to the TV library. But using separate volumes for `/downloads` and `/tv` will cause Sonarr to be very slow when moving files from the download folder to the TV library.

Explanation: On non Windows systems, Sonarr will always do a copy+delete instead of a normal file system move. (Reason according to a Sonarr developer: "This is to deal with some issues with mono in some file system combinations that lead to truncated files." [Source](https://forums.sonarr.tv/t/manual-import-move-files-actually-copied-then-deleted/23740/2)) This copy+delete is still fast if hard links can be created between those folders. But if the folders are on separate volumes (mounts), this is not possible. In that case Sonarr falls back to a normal (slow) file copy.

TLDR: If the "Read Only" flag in the TorrentBlackhole configuration is disabled (or if the UsenetBlackhole is used, which always moves the items) it's better to have the blackhole watch folder and the TV library on the same volume (mount).